packages/manager/modules/emailpro/src/domain/add/emailpro-domain-add.html

```diff
@@ -370,6 +378,15 @@ <h3 data-translate="emailpro_tab_TASKS_ADD_DOMAIN"></h3>
                     data-ng-bind="'emailpro_tab_domain_add_step3_configuration_' + model.srvParam | translate"
                 ></dd>
 
+                <dt
+                    data-ng-if="model.domainType !== nonOvhDomain"
```

Comment:
You have ovhDomain, nonOvhDomain and isOvhDomain. Maybe we can reuse or standardized to avoid getting lost between affirmation and double negation.

Reply:
`isOvhDomain` wasn't existing but I added it to make the code more unform